File: c/fibonacii.c
#include<stdio.h>
int main()
{
	int n;
	int a=0,b=1,c,i=0;
	printf("enter the num\n");
	scanf("%d",&n);
	printf("%d\n%d\n",a,b);
	for(i=0;i<=n;i++)
	{
		c=a+b;
		printf("%d\n",c);
		a=b;
		b=c;
	}
}

File: PYTHON/python_practice/fibonacii.py
def fibonaci(n,a,b):
	c=a+b
	return c	

n=int(input("enter the num"))
a,b=0,1
print(a,b)

for i in range(n+1):
	x=fibonaci(n,a,b)
	print(x)
	a=b
	b=x
